scripts/create_gitlab_snapshot.py

```diff
@@ -26,17 +27,26 @@
 log = logging.getLogger(__name__)
 
 
-def main():
+def main(argv):
+    import argparse
+    parser = argparse.ArgumentParser(description=__doc__)
```

Comment:
Add a short 1-2 line docstring to the top of the file so it is displayed when `--help` is used.